doc/guide/cockpit-http.xml

```diff
@@ -27,7 +27,15 @@ http = cockpit.http(options)
         <listitem><para>Connect to an address other than localhost. Must be a valid host name or IP address.
           To use this option you also must provide a port number.</para></listitem>
       </varlistentry>
-
+      <varlistentry>
+        <term><code>"port"</code></term>
```

Comment:
This is a bit confusing. Can I put a into the `endpoint` parameter of `cockpit.http()` *and* as `port` into the options object?

Reply:
I will try some variations since I found this doc confusing to begin with. :) In my successful use I left out the _endpoint_ and just used the _options_.

Reply:
You actually can put it in both places, but the port in `endpoint` wins over the port in `options`. I updated the documentation to point out that it's a fallback when not specified in `endpoint`, which hopefully clears it up.